docs/snippets/basic-external-secret.yaml

```diff
@@ -17,4 +17,7 @@ spec:
       version: provider-key-version
       property: provider-key-property
   dataFrom:
-  - key: remote-key-in-the-provider
+  - extract:
+      key: provider-key
+      version: provider-key-version
+      property: provider-key-property
```

Comment:
We can take the `property` from the basic snippet. It can be confusing for newcomers to see these fields that are only used in edge cases in some providers